src/macro-carousel/macro-carousel.js

```diff
@@ -1328,7 +1334,7 @@ value. Add CSS units to its value to avoid breaking the slides layout.`);
     this._slides = this._getSlides();
     this._slides.forEach(slide => {
       if (!slide.element.hasAttribute(ATTRS.STANDARD.TABINDEX)) {
-        slide.element.setAttribute(ATTRS.STANDARD.TABINDEX, -1);
+        slide.element.setAttribute(ATTRS.STANDARD.TABINDEX, '-1');
```

Comment:
We should probably apply the same changes to other `setAttribute` calls in the project

Reply:
I can't spot any other occurrences where we have to do type coercion